packages/devtools_app/lib/src/charts/chart_trace.dart

```diff
@@ -175,6 +174,12 @@ class AxisScale {
     return {'exponent': exponent, 'fraction': fraction};
   }
 
+  /// Produce a rounded or first number of the Y-axis scale and its
```

Comment:
I don't understand this comment. What does "The goal
  /// is to show high water mark (over 7) flips to 10's unit and
  /// it's exponent."
mean?

Reply:
/// Produce a whole number for the Y-axis scale and its unit using
  /// the exponent. Goal is to compute the range of values for min, max
  /// and exponent (our unit of measurement e.g., K, M, B, etc.). The axis
  /// labels are displayed in 1s, 10s and 100s e.g., 10M, 50M, 100M or
  /// 2000, 10K, 20K, 30K.
  /// 
  /// @param round if false, chunks the whole number to keep more available
  /// space above the max Y value highpoint to handle future bigger data
  /// values w/o having to rescale too quickly e.g., over 3e+3 displays:
  ///    10K, 20K, 30K, 40K, 50K
  /// This allows new data points >30K to be plotted w/o having to
  /// re-layout new Y-axis scale.